euclidlib/photo/__init__.py

```diff
@@ -1,5 +1,6 @@
 __all__ = [
     "angular_power_spectra",
+    "two_point_correlation_xi",
```

Comment:
This needs to match the actual function name below.

Reply:
Thank you for noticing, it's matching now.